php-checks/src/main/java/org/sonar/php/checks/SelfKeywordUsageCheck.java

```diff
@@ -83,13 +94,26 @@ private static boolean isFinalClass(ClassDeclarationTree tree) {
   }
 
   @Override
+  public void visitMethodDeclaration(MethodDeclarationTree tree) {
+    if (CheckUtils.hasModifier(tree.modifiers(), "private")) {
+      privateMethods.add(tree.name().text());
+    }
+
+    super.visitMethodDeclaration(tree);
+  }
+
+  @Override
   public void visitClassPropertyDeclaration(ClassPropertyDeclarationTree tree) {
+    if (tree.hasModifiers("private")) {
```

Comment:
@ivandalbosco this call we check that `tree` has 1 modifier, which is `private`.  Can `private` be combined with smth else?